internal/integration/helpers.go

```diff
@@ -258,7 +260,7 @@ func testClient(tb testing.TB, srv *server.Server) *http.Client {
 	}
 
 	return &http.Client{
-		Timeout:   5 * time.Second,
+		Timeout:   50 * time.Second,
```

Comment:
This seems like a really long timeout - can you explain more about the increase please?

Reply:
I have documented above this. For now it'll work, for long term, I can export *http.Client in *integration.Client so that the timeout can be customized for each test